dandi/cli/cmd_validate.py

```diff
@@ -40,23 +41,24 @@ def validate_bids(
     grouping="none",
 ):
     """Validate BIDS paths.
-
     Notes
     -----
-    Used from bash, eg:
-        dandi validate-bids /my/path
+    * Used from bash, eg:
+    dandi validate-bids /my/path
+    * Deprecated
     """
 
-    from ..validate import validate_bids as validate_bids_
-
-    validator_result = validate_bids_(  # Controller
-        *paths,
-        report=report,
-        report_path=report_path,
-        schema_version=schema,
+    warnings.warn(
+        "The `dandi validate-bids` command line interface is deprecated, you can use "
+        "`dandi validate` instead. Proceeding to parse the call to `dandi validate` now.",
+        DeprecationWarning,
     )
+    import inspect
 
-    _process_issues(validator_result, grouping)
+    from .cmd_validate import validate as validate_
+
+    print(inspect.signature(validate_))
+    validate_(paths, grouping=grouping)
```

Comment:
`validate` is a Click command object, not a normal function.  You need to invoke it as described at https://click.palletsprojects.com/en/8.1.x/advanced/#invoking-other-commands.

Reply:
this I think addresses the former point as well :)

Reply:
ok, I think I solved it, thank you.